src/MoeLens.sol

```diff
@@ -185,11 +192,12 @@ contract MoeLens {
 
     function getFarmDataAt(uint256 pid, address user) external view returns (Farm memory farm) {
         farm.pid = pid;
-        farm.isRewardable = _veMoe.isInTopPoolIds(pid);
-        farm.votesOnFarm = _veMoe.getVotes(pid);
+        farm.isTopPool = _veMoe.isInTopPoolIds(pid);
```

Comment:
is this refactor necessary? achieves same thing and breaks possible compatibility (who knows)

Reply:
I can update it back to the previous name, but now that there are 2 type of rewardable pools, it's a bit weird to keep it named `isRewardable`.
If we want to be extra cautious, I can use this value for both pool type, and add 2 bool: `isTopPool` and `isStaticPool` and if any is true, then `isRewardable` would be true?

Reply:
Not really a pool can also be nothing, eg, a pool that was previously added as a Votable (or static) and that is now removed will still be returned there (mainly to allow user to withdraw).
That's why it either needs an enumerable or 2 bools